src/metrikker/iatjenester.ts

```diff
@@ -10,24 +10,42 @@ interface IaTjenesteMetrikk {
     kilde: String;
     tjenesteMottakkelsesdato: String;
 }
+export enum IaTjenesteKilde {
+    SYKEFRAVÆRSSTATISTIKK = 'SYKEFRAVÆRSSTATISTIKK',
+    KALKULATOR = 'KALKULATOR',
+}
+const erMetrikkerSendtForKilde = (
+    orgnr: string | undefined,
+    sendteMetrikker: [TjenestePerOrgnr],
+    kilde: IaTjenesteKilde
+) => {
+    return sendteMetrikker.some(
+        (tjenestePerOrgnr) => tjenestePerOrgnr.orgnr === orgnr && tjenestePerOrgnr.kilde === kilde
+    );
+};
 
 export const erIaTjenesterMetrikkerSendtForBedrift = (
     orgnr: string | undefined,
-    sendteMetrikker: [string]
+    sendteMetrikker: [TjenestePerOrgnr],
+    kilde: IaTjenesteKilde = IaTjenesteKilde.SYKEFRAVÆRSSTATISTIKK
 ): boolean => {
     if (orgnr === undefined) {
         return true;
     } else {
-        return sendteMetrikker.includes(orgnr);
+        return erMetrikkerSendtForKilde(orgnr, sendteMetrikker, kilde);
     }
 };
 
 export const iaTjenesterMetrikkerErSendtForBedrift = (
```

Comment:
Hva er forskjellen mellom 
`erIaTjenesterMetrikkerSendtForBedrift` og 
`iaTjenesterMetrikkerErSendtForBedrift`? 😰

`iaTjenesterMetrikkerErSendtForBedrift` er e n veldig skummel funksjon, fordi den utgir seg for å være en enkel sjekk, men har en sideeffekt (pusher til `sendteMetrikker`). Den bør helst renames.

Reply:
Det er vel en forskjell mellom de 2 funksjonene, den ene er en `boolean` som bare forteller en ting. Mens den andre er en funksjon som GJØR virkelig en ting. 
Navngivning kanskje ikke er den beste, det kan jeg være enig i.